src/schema/objects/metadata.yaml

```diff
@@ -1797,6 +1875,16 @@ MRAcquisitionType:
   enum:
     - 2D
     - 3D
+MRAcquisitionType__mrs:
```

Comment:
Not sure it is worth making another metadata entry here when there is so much in common with `MRAcquisitionType`: probably better to make `MRAcquisitionType` more generic and add any extra detail (DICOM tag correspondance) when calling the macro for MRI metadata.

Reply:
I did it this way to allow `1D` to be a third option for MRS only. Is there another way to do this aside from this approach?

Reply:
Is this a valid value for DICOM Tag 0018, 0023 `MR Acquisition Type`? If not, where does this come from?

I would probably be okay with permitting `MRAcquisitionType` to be 1D/2D/3D in general, even though 1D doesn't make sense for MRI. We could put in a special warning in `rules.checks` if 1D is used outside the MRS context.

Reply:
1D does appear to be a [valid value](https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.13.4.html#table_C.8-87). And, yes, 1D only makes sense for MRS.

Reply:
I think 1D does make sense for MRI as well. There's even been some use of linescanning recently: https://pubmed.ncbi.nlm.nih.gov/34415208/

Reply:
I've resolved this by simply adding `"1D"` as a possible value for `MR Acquisition Type` in [ff370d7](https://github.com/bids-standard/bids-specification/pull/1377/commits/ff370d7c8df59b2e8a3691b8341069eafc229077).